contracts/account/Account.sol

```diff
@@ -5,6 +5,8 @@ pragma solidity ^0.8.20;
 import {ERC721Holder} from "@openzeppelin/contracts/token/ERC721/utils/ERC721Holder.sol";
 import {ERC1155Holder} from "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
 import {ERC7739Signer} from "../utils/cryptography/ERC7739Signer.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+import {Multicall} from "@openzeppelin/contracts/utils/Multicall.sol";
```

Comment:
I feel that adding multicall here is a long discussion. @frangio and I already had mixed feelings regarding the `delegateCall` in `executeUserOp` so I think Multicall may be too much.

Reply:
I don't think this is related to `executeUserOp`, but I'm against Multicall on the account, batches should be handled by some "executeBatch" function.

Reply:
Closed: using ERC-7821 instead